src/lib/GelatoOpsSDK.ts

```diff
@@ -152,6 +162,19 @@ export class GelatoOpsSDK {
     return taskReceipt;
   }
 
+  private async _requestAndStoreSignature() {
+    this._signature = await this._signer.signMessage(this._signatureMessage);
+    const token = Buffer.from(
+      JSON.stringify({
+        signature: this._signature,
+        message: this._signatureMessage,
+      })
+    ).toString("base64");
+
+    // Set Axios headers
+    axios.defaults.headers.common["Authorization"] = `Bearer ${token}`;
```

Comment:
if the SDK is included in another project that also use axios, won't that potentially break the project requests? (if they also use bearer token)
I think we could instantiate a local version of axios in the sdk to get around this